magefile.go

```diff
@@ -95,6 +97,17 @@ func Spec() error {
 	return sh.Run(swagCommand, "init", "-g", "cmd/main.go", "--pd", "-o", "doc", "-ot", "yaml")
 }
 
+// Integration runs an integration test. Note: the ssi-service must be running for this to work
+func Integration(testName string) error {
+	switch testName {
```

Comment:
this is good for now but I wonder if we can do it so we don't need to have 'known' tests in here, we can do some sort of lookup and run the test if it's found